pkg/node/config_defaults.go

```diff
@@ -40,7 +40,7 @@ var DefaultRequesterConfig = RequesterConfigParams{
 	OverAskForBidsFactor:               3,
 
 	MinBacalhauVersion: models.BuildVersionInfo{
-		Major: "0", Minor: "3", GitVersion: "v0.3.26",
+		Major: "1", Minor: "0", GitVersion: "1.0.4",
```

Comment:
Does the Git version definitely not need a `v` in front of it??

Reply:
oh it is needed. Thats what you get when you do a late night change :S
Fixed in https://github.com/bacalhau-project/bacalhau/pull/2843/commits/0d8023d9f9786a12a89d6e036de0e24b2112313f